pyproject.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 include = ["profiles/*"]
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.11"
+python = ">=3.7.1,<4.0"
```

Comment:
is `<4.0` really the best way to capture this? There's no guarantee of a `4.0` release, and it seems like the next releases are `3.12` and `3.13`:
https://docs.python.org/3.12/whatsnew/3.12.html

Reply:
Agreed, updated to 3.12